### ISSUE TITLE
Multiple select inputs have no label.

### ISSUE THREAD

**Issue by JohnDBowne** [formr/Formr#142]
Hi, 
I'm using 1.4.2.  I recently upgraded from 1.2.4, because of a entry in the changelogs that suggested that multiple selects would now display a label. 
```
		$showDataForm = new Formr\Formr('bootstrap3');
		echo $showDataForm->form_open();

		echo 	'<div class="row">';
		echo 		'<div class="col-sm-3">';
		echo 			$showDataForm->select_multiple('showData_select_period[]','Period(s)','','','onchange="this.form.submit()"','',$_SESSION['showData_periodList'],$periodSelectArray);
		echo 		'</div>';
		echo 	'</div>';
		echo $showDataForm->form_close();	

```
This yields the following in Chrome:
```
<div id="_showData_select_period" class="form-group"><label for="showData_select_period">
</label>
<select name="showData_select_period[]" multiple="" id="showData_select_period[]" class=" form-control" onchange="this.form.submit()">
			<option value="201607">July 2016</option>
			<option value="201608" selected="">August 2016</option>
                 ... shortened ...
			<option value="202110">October 2021</option>
			<option value="202111">November 2021</option>
			<option value="202112">December 2021</option>
	</select>
</div>
```
If I remove the '[]' from the showData_select_period[] in the above code, the label will appear. But, of course, I can no longer retrieve the data that I need.

Thanks,
    johnbo

**Comment by timgavin** [comment]
This is working as intended for me.

```php
$options = [
    'steak' => 'Steak',
    'chicken' => 'Chicken'
];

$form->select_multiple('food[]', 'Food', '', '', '', '', '', $options);
```

Produces the following HTML
```html
<label for="food" class="form-label">Food</label>
<select name="food[]" multiple id="food[]" class=" form-control" >
    <option value="steak">Steak</option>
    <option value="chicken">Chicken</option>
</select>
```

Upon submission
```php
if($form->submitted()) {
    $form->printr('POST');
}
```

Produces the array
```
Array
(
    [food] => Array
        (
            [0] => steak
            [1] => chicken
        )

    [submit] => 
)
```

As you can see my HTML is a bit different from yours; mine says `multiple` where yours says `multiple=""`. Are you sure you're testing this with 1.4.2?

Also, you no longer have to add `echo` to your elements. Got rid of that baggage a long time ago :)

**Comment by JohnDBowne** [comment]
I've checked the version number in the class.formr.php it shows Formr (1.4.2).

I've dropped some of the other bits in my call to make it look like yours and simplified the label text, but I'm still not seeing the label text show up within Chrome. 

And thanks for the reminder on the echo stuff.

```
$showDataForm = new Formr\Formr('bootstrap3');
echo $showDataForm->form_open();

echo  '<div class="row">';
echo    '<div class="col-sm-3">';
             $showDataForm->select_multiple('showData_select_period[]','Period','','','','','',$periodSelectArray);
echo    '</div>';
echo  '</div>';

```
```
<form action="/index.php" id="formr" method="post" accept-charset="utf-8">
<div class="row"><div class="col-sm-3">
<div id="_showData_select_period" class="form-group"><label for="showData_select_period">
</label>
<select name="showData_select_period[]" multiple="" id="showData_select_period[]" class=" form-control">
			<option value="201607">July 2016</option>
			<option value="201608">August 2016</option>
        ... shortened...
			<option value="202112">December 2021</option>
	</select>
</div>
</div></div>
</form>
```
If I drop the array syntax from the call:
`$showDataForm->select_multiple('showData_select_period','Period','','','','','',$periodSelectArray);`

I get the following output:

```
<form action="/index.php" id="formr" method="post" accept-charset="utf-8">
<div class="row"><div class="col-sm-3">
<div id="_showData_select_period" class="form-group">
<label class="control-label" for="showData_select_period">Period</label>
<select name="showData_select_period" multiple="" id="showData_select_period" class=" form-control">
			<option value="201607">July 2016</option>
			<option value="201608">August 2016</option>
        ... shortened...
			<option value="202112">December 2021</option>
	</select>
</div>
</div></div>
</form>
```
Thanks,
    johnbo

**Comment by timgavin** [comment]
Sorry, I didn't see that you were using Bootstrap 3 in your first example, and this appears to be an issue with the Bootstrap 3 wrapper, and will be fixed in 1.4.3

It's an easy fix if you don't want to wait. Open `lib/wrappers/bootstrap.php`, and on line 414 change this
```php
if (substr($data['name'], -1) == ']') {
```

to this
```php
if (substr($data['name'], -1) == ']' && ($data['type'] == 'checkbox' || $data['type'] == 'radio')) {
```
